### PR TITLE
update existing SIMDs with their current status

### PR DESCRIPTION
**Pull request by Benhawkins18**
This PR updates all SIMDs to a more current status and makes changes to the linter to disallow draft as a valid status and adds living and stale as new accepted status.

### CODE REVIEW

**Comment by Benhawkins18** [comment]
@jacobcreech linters are happy and this PR should be ready to go. Awaiting your review

File: .github/linter/customRules.ts [hunk 1]
```diff
@@ -281,13 +282,14 @@ export const metadataStatusIsValid = {
...
     const validStatus = [
       "Idea",
-      "Draft", 
       "Review",
       "Accepted",
       "Stagnant",
       "Withdrawn",
       "Implemented",
       "Activated",
+      "Stale",
```
Comment (jacobcreech) [inline .github/linter/customRules.ts:291]: What's the difference between `Stagnant` and `Stale`? We have `Stagnant` listed in simd-1

Comment (Benhawkins18) [reply .github/linter/customRules.ts:291]: ahh yes, good call

Comment (Benhawkins18) [reply .github/linter/customRules.ts:291]: fixed

**Review by jacobcreech — APPROVED** [review]

**Pull request merged**